public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Bowling Scorecard</title>
+</head>
+<body>
+  <div id='container'>
```

Comment:
Like the use of a container div. Its good to encase all your code in a div so that it can be manipulated easier.